### ISSUE TITLE
Error in getStoredFile!

### ISSUE THREAD

**Issue by nickname110** [l3nz/ari4java#69]
Fail to retrieve stored recording!
```
com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'RIFF$n  WAVEfmt': was expecting ('true', 'false' or 'null')
 at [Source: RIFF$n  WAVEfmt      @  �>    data n                                        ��  ��  ��    ��  � �� ��� � �����t�T�t���T�������d�$�$��4�$�������d�������$�d�d�$�����D�������$���������������$�����$���$�����������d���$���d�d�t�T���<�����$�����4���<� ,�,�4��T���l���� ��L������\\\�\��\���\\\�\�\���,��,,\\�����������\��ll���tH L�t�t���T�L�� ���t�����T�t�4�4�d���������d�$�����d�d�d�4���������������$�d�$�$�������$�$���$�������$�d�T���������D��t�4�t��t��,� P  ���T���4���$L�,l� 4� ����l��l��t\\��\�\\�����������\\\\\���\\lD�L�Lll�D` H ��� � � ,d� \�4���������H ����0 ��� x ��\���<���� ����T���$���4�����T���d���d�$�d�$���$�������t�T�������������d���t�����d�������d�4�T�����T�������$�T�������H D� 4�\���� �l����4��� $�L���l�l����������l������,L�L,�l�L,�,L��,���\��\�,L�,��L����Th 8 � x � � � �   <�T������t��  ���l�����t�L�X ����������d��d�t�T���T�t�t��������4���t���d�$���$�������$���t�t�t���T�T��T������\�dtllP  ����t���� �Ltp ��l����  4,�����������T���d���d���$��������<���4�4�����x Dl�t� P � � �\,�L�����\\��\�\\\�\�����\��\\\���\\��ll�\��l,���L��� ` L�L�` @ ��<�P |�4�t���T����������t���������d�d�d���d�d�����d���d�$�d���d���d�d�$�$���$���d�������d�d���$����d�4�t�\�������������$�����t�T����$�d���T�4�l����L�t�t���t�������H � d�TtT�lL�l��\��\��\�����\���\������\\\l��,l�\\�p ���t���<�� DT����|�T����t��|���4�T���t�����t�4���4��t�T�$�������$���d�$�$�$�������$���$���D�����d�$���D���d�D���$���d�d�d�D�D�������d��$�������$�$�$�$�$�d�4������L�<�������T�� d�����,�T���` D���� H ��� ,��l���� �L,����,�����D����t,��L��L�����T� ,�l,T��   ��\�� $�  ����d�$���$�$�����$���$�������$���d�$�������������$�������$�d�d�D�����D�d�����D�D�D���D�$���$�����d���D���������D���D�����d�����$���$�$�d���d���t��t�����4���t� 4x H �T����� ,��,x P ����` t�,�X ���dD���L\��LLl�,��L�\,lL���,��l� x Tt,L,�� ����t�` 8 � �������t�������T�T�������4�$���4�d�T�t�����T���4��������$���d�����$�d���$���d���$�$���������$�d�d���d�d�����$���d�$���d�����4�t���$�$���$�$�4����\�T�����T���|���|�@ X � 0 \�����H 4L�L�t�D�����l��lL��L\�\��\L���l�,,��LL,\\��������\� \�L���@ �4d� ( ����������@ � x ,�����$���T�����������T�d�����T�d�$�T�t�t�T���4�t���T�T�$�����d�������$�����$�$�d�d���d�$���$�d�����$���4�����d�����$���4������4��d�d�������\���� �� ����0 � 4t�l��X  � X D��l�$� �t�DlL����\�\���\���L�l�L�l���$��� � � �� P L�T���������( ��4���������$�$�T�t����4���d�d�d�d���d���$�$�����d�����d�4�����d���T�$���������T�t�����$���d�$�����4��t����� ������������4�@ � � @ ��t�������\�� D  ( � h <�\�tL�����\������\\�\��\\\���\��,��������\�� �\\����L,���L��� @ ����x ` ��,�����$���T���t���t�d�����d�d���������4������$�����������������������4���d�d���d�������$�$�����d�d��������4�4��$�4�4��$$d�@ � � l���l�� ���,�Tx t�l�,�,�td�l\���\���\����\������\����\L,\\\���l����,X ��� Dx � �  ��\���t�T������4���T�����������������d���$���������������������t�4�T�4�����d���$���$���4�\� � �������t���� � �� � t� ��X d�l\l� 4���,L,L�dL�����L��������\���ll��\\\���x @ �l�Ll,d� � � ����T�   � |�T����p |���������h � � ��T�t�$�������t�����$���������$�$�d�d�$�����d�4�4������d���d�$�$�������$���d�d���$�����d�$�$�d�����t�<� � L�<�t�t�|�\�L�����,L�� � � �L�t� � �� ��D�L����\�,�����,,,�,��L��Ltl�4L���������� DT8 L�T�4�������<�<�T�������$���d�d�d��������T�d�$����d�$�����d�$�$�$���$�����d���d���$�����$�d�$�������$���d�d�����$�d���������d���d���d���������T�������4������t�4���������t�T�T�����8 4� l���l�@ ���� ����l�,�\L�����L��\�\����\���Tt��\,,���lL,4��T���d�d�����d�������$���$���������$�����$���$�����������t�T���4���d�$���d�$���d�$�d���T�l���������$�������d�����d�T���������$���d���������t�����$�������T������� $l���h ��4�l����l�,�\\\�\���\���\\�\���\����\\��\����L\\���4���� ��,$dtp 4�����t � � ��L����������t���d�d�$�$�d�$�����d�d�d���$�����d�����$�d�$�����$�d���$�d���$�������d�$�d���d���$�d�d����4���$�$�d�$���T�t�, ���������t�L��l�t,��\l,llL\����\l�,��\�����\\�\��\�����������\�\��\�������,l,�L����4�������t������4�d�����$�����d�$�����d�����$�$�d���$�����t����������������t�T����t���T�t�t�\���� $��t�����T���������$t ��t������ ,�dL� Td��,���,����l����l\��\���\\\�\�\��\������\��$� � ,�,L�� \�( � l������������L����      ���4�t�t��t�  ��T�T���t�t�T���,�t���T���d�d�T���4�4���4���$�d���$�$��������t�����d���$�d���$�t���������$�$���T�����dx  ��4���4�4�\�p DX ����H x   $�lll��t� � DD����l\�\����\����ll��,�Dt�,�l�,��\��� dL���\�D� ��\�T��l�� �����������  � �  ��4���t���L�@ ��t����d�������d�$�d�����d���$���d�$�d�$�d�$���$�������$�����$���������d�D�d���d���T�T�T�������d�$���d�d�����4����$�����4�<�|�� � ��t���d����L�� H <� �<�`   � � DtP h D$4����L���ll�,�\���������,l�\\�L� ����,�T4tl�t�d�������d�T���$�$���$��������l�T��t�T�����$�������������������d�$���$���$�d�������d���$�$�$�d�d���D�D�$�D�D�d�d�d�$�$���d�������d�d�d�$��4�����d�4���d�������  � T� p <�\���� � 4TT� D� $���\\�ll����l,�\��l�L���,,\�L4d���,�,�T� ( X d� �� L�����T���d������$��d�d���$�$�$���$�$�������$�������d�d�$���d�d���d�����d�d�d�$���d�d���d�d�d���$�����$�d�$�$�d�$�$���$�������d�$�����D���d���d���T�����$�������������� � ����  ����4�8 � tll� � ,4   � l,,,T�� 8 �� T�l��,lL��\\\��\��\\��L�\��\����Td$���� ������l���|�����T���d�$�d�d�����T�d�����T�����$�d�$�$�����$�����$�d�$�$�����d���d���$�$�d�d�����$�$�$�d�$���d�������4���l���4���$�d�$�$�d������������$�$�����4�L�����|���t�,���  <���� � Ll�L��l�\�,���Ll��,LL�\��\����\��\��\���� x ��\��Dd�l�� ��4�t�t�t�h t� x l�t�4���t���t�����T�T�d��$�d��t����������d�$���$���d�$�����d�������������d���$�$���$���d�����d�������t���$�$���$�$�$�T�t�����L�t�����<���� ��L� ��� d4,�\�\�\\�\\�\�L�,\�\\l��\�\�lL��\��\�����l�,,L\\���,\�\Lt� ��� � D�   ,�t��T�t�t�4�l�  ,�����������������d�d�d�d���$�d�d�$�������d���d���d���D�d�$�����$�$���$�d���$�d���4���L�t�$���d�d�d���d�����������T�d�$����8 �� ���T�����T�������������l�` ����,��l����\���������������\�l,�,� H ��� ��,�,���L�( P ������������d�d���d�����������d���$�d�����$�d�����$���$���4�������4���d�������������$���d�$�������d���$�����$�����D�����������$����$�d�����D���$���d���$�4�$�d���d�����$�����t���t���L���\�\�� LL�L�� �tTl��L��\\\\�\�����,����\��\\��t����h L��D0 ��\������tL,4����������� X �� 4�4����T��  ��T���t���d�����4�����������������$�����d�$���d���d�$���$���d�$���D���������������d�������4�d���d���d�d������0 ��������L��� ���,l�,t�TLL���l�,�,\L�$�L��\��L�����\���\\\\�����\���������\Ll�� �����\������ ��������T�$���t�t���d�$���$���$�d���$�$�d�����$�$�$�T���d�d�$���$���T����4����d�����d���d���$�d�d�d�4���������$�d���$�t���� � ��������4�������� � p � ` ������� ����l4L,������L�L��\���\�����\��\�\�\����\��,�L�L��d� ��� � � � T� �T��������\� ��L���4������4���4���������T�$�d������4���$���d�������������$�$�$�������T������T��d�d�T�d�d�����$�d�d�������T�d�$�d���$����t���t����4�,�� �,lD4���l���l�L���\��\��������\�\���\���\�\�\\\\l���llL,�l�D4��LlL���,d4,�l�,�� ��������t������T�������$�����d������d���d���d���d���$�������d�d�����d�����d�������4�4�t���t�T�������d���$�$�����t�t��������������t�\�D����� ��h l��l��L��L,���\����\����\�\�\\�\���\������������\�,4$��` � �t�Tx 0 ����T�<�� ` � ����������$�T�4�������T�$�����d�T��������t�����d����$���d�d�T���T�d�d����d�4�������$�������d���T�l�0 l�T�����$��������DH ����d�4������\�$4�8 ,�t��� 4�,L���� 4��������\�\\����\����\�����\l�4� l,,�L�x ,���<�t�  T�� ���������d��|���������,���t�4�������  t�t�����$�$���$���d�����d���d���d�������������d�$�$����������d�d���d�d���d�����������4�����d�$���T���\�` ( ���������t�����` t�D8 �����\�x 4�,�4tT� ���\���l�\�\����l���l�LL�L���l,\�l�x P d,�\\�ll� h ��� p � d� ������$���d������T�T���$�$�$�d��������$�����D�$���$�����$���d���d�����d�������$�$�����������d�T���$���$�����$�d�������t��L���d�d�d�����d�����\���L���4�T���<��l���tx �����t��,x � ,t� ,l,�lL����L�,����l��,�,�\������,,� H D$��l��������������� 0 x ��t�������d�����T���t���4���$�����$����T���4�4�T���d�$�d���$�����d�$�d�d�$�$���d�$�����d���d�����$�$�����$�������d�d�$�$�$���l�t�$�������$���������T���������t�t�����Td�TH X ��4,L� �l,,�L�l��,\�L���\\��\\�\,\\��\��\�\ll���\���,���\����4� 4dd4p � @ L���������������������d�������������d���d�������d�����$�d�d�d���T�t��$�$�$�������d���$�d���$�$���������������$���$�����4�����4��������<���P �4� � 4$L�\\���,���\\�\������\�<	����\\\\\\�\\\\���L��,�l� h X ���L�� ��<�����<�X p \�t�t����\���4�T������4���t���������������( x   ����T���T�������������������� � \���������������P ����D����4�  ���\�L` ����\�����\��\\��\\�\\\����\\\\\�\���\�������L��\\���T��$T� ����������� ��  ��( 8 ������( ����4�$���$���$�����������$���$�$�����$�$���d�d�$�d���$���d�������$�d�������d�$�����t��� ���T�L���� T�L����� �,���,d�D  L�� �L���d� d� ����L��,��,�\\\����l,\���\�\\\\\\�\����������,�DH �� T,dx ,���t���������������T�����4�$�$���$�$�d�������$�$���d�����4�t���4������������d�d�d���$�d�d�t��4���T���<�T����t�4���� |���,��t����@ 8 H � $��t��4�4������ �� x x ( h ��lL,L\��\�����\\�l�\��\�������l�4�L������� ������<�� d��|�����t�����4�<��� ��������T�T�t�t��4�4�t���T�T�T�����������4�$���4�$���������d�d�������d�����������������d�d�d�d�T�T�������d�d�d�������t���� �4���t�,�\�4ll�l��\���\\�\�\\��\����\�l�\\\�\���\����t��,� X h � ����Lt� ,����  p ����� �����������������d�T�4���d���d�d�d���d�$���$�d�$�$�������d���$�����$�d�������$�D���$���d�d�����d�$�T���d�����$�d�$�������� ���t���t�T�,�l������ ` \����t���4�����p ����<��l�� �������t� L�ll�����\�\�����\\\\�\�l��\��<	\�,�������,,4������l�X �����������t�4������������T������l�|�t�T��d�4�<����� ���������T�T��t�t�4�$�$�����������������X �T�������$�d�������t���4�$���������L��������������l�lLll���l������\\����\���\�������\�lL,��\���l�,�\��,�D` |�@ H � ��Th ��t���4����������L����T�����������T���d����������������������,������T�T�t����4���������t��� \����t��t������ ����T  � l\\\�\��ll�\��\�����\��������\�\�\�\\���\�������,l\\�L�,LL\�l�� � $�d� ( \�,�����T�T�t��,�t�4���������d�������T���$�$���t�t����������t�����t���$�d�������T���$���$���$�d������t�������d���$�������T�����t�T�d�$�d���<� h  \���4����� �L,L� ��� � � ,������\����\��\���������l�d,��L�� ��� $���4��t���<���@ � � � � ( ������t���T�����t����t����������������$�������d�d�$�d���d�$�������d�d�$�$�$�����D�D�D���$�������$�4���t�d���d���$�������T�4�t�T�t���4�4���  P @ 4���t��T�t�����<�t�4�4�����������d,,�Lt� � �����,L��t��� � D������$DllD� �����4�L�p � �� (  ������4�����h � � <�4�d�����d���T���4�������d�d�d�T�t�$�d�$�����$�$�d���d�$���D�D���D�������������D�D�����D�D�D�����$�D�$�D���D�������$���d���������d�����������4�4����t�$�����d���t��������4�d�d�d�d���t�t�L�,�4���,�l���x �Tll�����\��lL��\���\���\��l�D� D�L��� ������|�<�������\���4��$�d���$������4���t�4����d��4�������d�$�����d�$�d�����d�$�������d�d���$�d���d���d�d���$�������d���D���D���d���d�4�����d�������d�����������t�T�d�d�d�$���4�<������������|�,���� �4x ��P 0 ��� t���� L�d�� �,����,�\,��L�lll�����l��` ����� D��,�H   |�,�� � ��� ������\�l�T���H �  ������4��������t������4����4�4���4��t���d�$�d�������t���d���t�����t�������4�������t��������T�����x  |���<�t�T�\�,�l�l�(   ,���L���,�$��,�L������,�l��l�\���l��\��\��,lll���\��\�L�������\�����d�d0 <�t�4���d�����4�t��T�����������������4�d���d�$�$�d�������������d���$���������$�������d�$�$���4�$���d�����t����d�$�d���$�����4�4�$�������d���T���4���������$�$�4�t���,���L�����4���� d�l�,�� �LL�\���Ll��\��\���\�\�\�\���\�������\LLL� � X � �L�@ \�\�L� D�,�� p $� ` �� ������T������������4�d�����$�d�$���d�d�����������$���t���T���������$�����d�������t���T�4��$�������������L���d�����$�������|�� �8  �� 4� 4D���$L��d�,,l�\��\���l����� � � ,l�,,L0 �������$l�L������� ��� D� 0 ��T�����t�������������$�d�d���d�����$�d�������$�������$�$���D�D���D���D�D���$���D�D�$�$�d���$���D�D�������$���D���D�D�����4�4���$�����d�$���$�����t���������T�������������t�4���d�$�t�4�t�<���    p ` ����� d�$tD��� TL,���\��\����l����l��lx ���L�D � ��,�L����� ,���t�$�d���������$�$�����������d���d�$�d���d���������d���d�$�������d�d�$�����d�����d�d�������$���d�d�$�����d�$���$�����d�d���$�$�d���d�����d�d���d����d���d���������4�\�,���T�4���� P ���D4�l����L������L,�L��l����,��L�\\��P ��������� ������$4� P H ����������,�����������t����t�t�����<�|�T�t�����T����t���4���$�d�����$�d���������$�d�$���d�d���������d�$���$�������$�d�������d�$�d�������d�������4�����d�4���t��( � � � x l���,����0 � � 4����d� �  4L� ������,�L��,�\\\,��L��\����\��������,�� ��� d� � � ��|�4��t�����t�4�d�d�$�$�d���$�����������d�d���$�d�������$�d�$�$�$�$�������d���d�����$�$�$�d�����$�����d���d�d�$�$�������4�T�����d�d�$�$�d�����l���t�$�$������T���@   0  ,�@ TD�� � $� D� �l�,��,��t��\l��l���\��\L����,�������� �L����4 $�L�l�d� ��������t�L���������t�4�������4�d�����$�$�������d���d�$�$���$�d�d���d�$�d�d�����$�$�d�$���$�d���$�d�d�����d���d���$�������$�����D�$�����d���d�d���d�d�����$�$���t���� |�4�T���D���l�����\��\\\����\��\����\\\���\\\\\����\�\,��L�l���,����T�4�|�������������t���������4�4�������������d�$�$�d�4�$���$���������d�t���$���d�$���t�����4�$��������4���<��������T����4�������l�����t���T�T���,�P T��\�|�L���� ( p � � �����Lll����\�\\\�ll��\���\������\\�����,�����LD����H ��� ��D���4�T���4�� � ��@ P ��<�l�������$�d����$���$�d���d���$���������d�$���$���d�$���$�������d�d�����$�d���d���������$���d��T��T�d�����$�$�d���T���t����������t��\�� ` ����������0 ��$����� � ,�\�L\�������\���L\�����L��\L\�l��� t���l��� p p � 4T�lL l�T�����������������T���T�������T����������$���d�$���$�t�d���$�$���d�����$�$�d�d�d�����$���$���$�������$�$���d�����d�$������d�����d���$�T�t��������������|�� � ��t�l�����<�T���� P T��Ld���\l\L��������l������\�\�\����\�L��\��l\�����l� l���\�� � ��x 8 �������� h ��l������T��4�4�T�����t�t������4�t�4���4���T�4�������T�����t����d����������$����X � T�4�������d�d�T�t�\�\���������  H �D0 ��� � � p ����D���,�l�l�d���\\���\�\�l����\�\����l���h L���,���,�4 ��������d���$���4�4�������t������4�����������4�t�T�������T���t���d�d�$���d�d���T�$�d�$�d���$�$�����d�����d�$���$�����4�4�d���t�T�������$�$���d�4�������X ����<�<�p @ � ,\�l�L\\�\\���\����\�\�\��\�\\��������\$�l�<D�,����<���,t|���� ����   X ���\x � @  ��<�4�����D�|�����d�����������������������0�������|������\�,�t�<�|��d�t�|�L����,�����������t�������(�h���h����,�` ��t���<�T�������� ��������t����< ��|<4Hh���X���4�D,�$��t,4�\������ P0��T�DL�� L����,�\� �����l|  ���(�h�t�����  ���<���8�����t��<�D�����|�����l�4������\�����L�8�x�x���X�������D����l���t���D���|�������D�������|���T�L�4�,���������������������������X�  ( ������p �D��$,�,���T��P�`�T�D<�,�\TD<|���t������,,�����(�������d�4L ��X���l�$�������� H ��T�<�t�������$�,���(�������(�����\���L��<��������|���$�4���D��������l�d�\�������@����� ���`�����T�|�D�������,�4�<�������|�����|�d�l�������<���L�x�h�8���x�X�� 0 |�� �����L�l��� � 4� ��l��4�����������|�Th����X�����\��|D�$4\$���<��p ��P�x0��PD�� ��D$4,|Ld����$���T�����������t�|�4�$���,�������P���<�<�����|������$����������l�t�����\�d���|�D����������(���(���h����������D�L�d�<������|���$��������,�d��������4���������h����<���L����0�����P�    ���D�� |� 0 X T� � �t|�P��� �4���L�D�4dT�Dl���4�4��4���44� dD ����L�������\���P l��������<�����l���������x�h�h�8���L�������|���������d����D�t�����������,�����L��<�<�t�������0���p��������������\�  ����������t���T������t,�<<����p�p��<<���T�$\�D|,�����t,ld�XXX�d�,��\l�|�4�<tl<�T��4<<��` ��� @ ��|�� � @ ��l���t�������������$��������D�|�4�<�T�������0�P��������d��<���������������<���|�D�\���t�|���L���l���������(���������� 4���|���D�d�L\TD�$$� ����D\\���h�H�dl�4����L$,tD�����|�T��`����D<t���D4�\�����Dl���t�� � h(� � ��d�� � �4��P � p d�����������0 ��4�l���������������,���$����������������4�,�\�������<���p�L � ��������������H� p ��T��� ���\���� ��<��DL�\�\d�������|�4�d$���<	�\�\\��\������,llL\�\���L�,L\L� $� ����\� P 0 � � � ��T���T�T�t���t���d�����d�d�$�$���$�d�������d�$�d�d�$�����������$�$�d�����$�d�����d�$�$�d���T������( \�4������������ � ` ��t�T�|���T ` � ��d� |�  0 d,���� tL��\����$��\\���T����l�T\����d��\���L���$L��� ����|����|��������,�L�( p���t�����4�4��������������4�������\���L������������|�<���,���4�$��|���\����,�l���\���T���������<�� �4h ������l�4�T�����|����d����������� D��,l�������4�d$t�����|,<T��,T�d��������L��T��L�t���$� � LD�|�dD�d�0�\������� D ( (�\��`   ��0 ,� � � l�x� ���4��8�����P�T�t�4��� �������<�p����������������X�8�L������������������� `����h�����\�   � ���$� 8�� P�@8Hh��� p pd0xP�D(`d���X���`l���`�Pt�����p� 	@	�	��p���	�	�	� ��\�p`�(�X4�� �8 �|�T ��T��� ���@ ����X �� p���$�� P �h(�<��h�  $�������� ��������(�x�`� �����0�P�X�������������@����8�����<�L� � ���|��x��H�� �H����H�@ ��� �00�D�����H\������H������H(0H��(
|\�L�<������x�0� �(���� (��H `����x�� \ (H�\�@�� 0�� ��  d�����������  D���P�@�� ������p�x�p�������<��������������X���<�����X�x����������0���������������8���(���������X�D���`���(�� �L� ��@ �� L���xH��x$@(��`��` �`��H��h0�����(pP�����H�@�����h��` p��  �����H�H������8������h�8� �@�@�`�����P���0���P�T��� ���  t�D�����p�����p��h���H�(���h������`�����,���������� ����������������������  ��(p�`�������x���0 �  �� `�<��<�48� < 8  h�0��@��<�<h�DDP� X�t<0��@��  ` 8���p �������p�x  ������ ���� �� X ��<��Xx�` � � � H�� l����@h�����`�����@ H�@�������H�8�x��0�0�,�X��������������p���P�8���p�������8��������p��� �(�`���  X�X�������,���@�l�@�|�p�����0�8�������� ��� ���h���P��X��p���t����x ��8	���0(pT`�x
@�l@����P��x�� �P�h������� �X���h0�� �� �(P���<8��D�@t����X����(�x�����H������������ �H���@�����������H���������� �������h����� ���`8���8���P�������������  ����������0���  �X����@�8�`����\���������� T��������TP�8����x	8
�
� ��(�����x�8� ���0  ��@� ��hp�00��
#H�����pΰ�P� ��H
��h� ! �	(�h���������X�ٸ�0̠�X�8� ��0hX�h(Hx�������p���(����h�(Ш���<�  �
HH  H�$�t�`�X�X�����P&p+`�@�p�p`�����(������ژ�0�``( `����� �(���(��h�(�(������
 `� P�h���8��¸���8���`�#�P�p�T�
�� � �p��،ňؐ���(��
�@@�@�	� (�@���hԨ�(�����H�(�� �	 
��$ (��#h		`�� � �`Ԡ�`�P�X$h"���� �P����`�`����߸�0����X��P!�h0�
0�8���8�x���8�(�h�� ��	�`���`�@ ` 
��`�� �h����h���8�X��ߘ�����0�`  000p������
 ��H��h�
� ��`(�(��h��(�h�(�h����������`�  `
�`
h� � P���x���X��������(���x��	��``p��0�� (�( ����h������P�0�� ������H����"�hh�x� ��� �����������`���p�0�0������H���� 8(�X!�h�p�  ����@  �P�������x���d���8� �`�p
���%�'*(��D������P��� �`�P �8
������� �p���`�� 0XX$�*H'(+("�Xx
X������������ �`��`��H
�P0 �00� (�x�8������8���x���0� 
HXX�	� P���x�(��d��� ��������"p�H(�
���������h��d�D��� �� 
 �p���h�X�0�D���4�\�����H(���� �0���p��������x�h
�0��`�P��P�P�h�(��(�Hp��X � �,�L�D���,�@���l�
pP�00
@��h���8����X�����`	��p�� ���h�(�������4$
8���(	@���� ���`��(�� �HX�H� ��H�� ��p�����(��� � h
��
��\�����0���P��@�P�(�@�������p0��H�x�������X���P�0 �X�����h��(�h��������`
(p�p�P�����(��0�  ����`� ��� � ��D�
@<t	����8�x�(������P�����P�p����H�h�(�(�(�(�X������  �
���P�; line: 1, column: 16]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1581)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:533)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:2462)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:1621)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:689)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:3776)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3721)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2726)
	at ch.loway.oss.ari4java.tools.BaseAriAction.deserializeJson(BaseAriAction.java:132)
	at ch.loway.oss.ari4java.generated.ari_2_0_0.actions.ActionRecordings_impl_ari_2_0_0.getStoredFile(ActionRecordings_impl_ari_2_0_0.java:322)
	at socketio.listeners.FileRecordRequest.onData(FileRecordRequest.java:33)
	at socketio.listeners.FileRecordRequest.onData(FileRecordRequest.java:15)
	at com.corundumstudio.socketio.namespace.Namespace.onEvent(Namespace.java:134)
	at com.corundumstudio.socketio.handler.PacketListener.onPacket(PacketListener.java:103)
	at com.corundumstudio.socketio.handler.InPacketHandler.channelRead0(InPacketHandler.java:92)
	at com.corundumstudio.socketio.handler.InPacketHandler.channelRead0(InPacketHandler.java:36)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:187)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:846)
	at com.corundumstudio.socketio.transport.WebSocketTransport.channelRead(WebSocketTransport.java:94)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at com.corundumstudio.socketio.transport.PollingTransport.channelRead(PollingTransport.java:109)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at com.corundumstudio.socketio.handler.AuthorizeHandler.channelRead(AuthorizeHandler.java:132)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:108)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:182)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:308)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:294)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:846)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:130)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by edgardlopes** [comment]
I had the same problem. But the output was a little bit different:

`com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'RIFFd': was expecting ('true', 'false' or 'null')
 at [Source: java.io.StringReader@5185cd61; line: 1, column: 6]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1702)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:558)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:2839)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:1903)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:749)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:3850)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3799)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2858)
	at ch.loway.oss.ari4java.tools.BaseAriAction.deserializeJson(BaseAriAction.java:132)
	at ch.loway.oss.ari4java.tools.AriAsyncHandler.handleResponse(AriAsyncHandler.java:37)
	at ch.loway.oss.ari4java.tools.AriAsyncHandler.onSuccess(AriAsyncHandler.java:65)
	at ch.loway.oss.ari4java.tools.http.NettyHttpClient$3$1.operationComplete(NettyHttpClient.java:246)
	at ch.loway.oss.ari4java.tools.http.NettyHttpClient$3$1.operationComplete(NettyHttpClient.java:236)
	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:680)
	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:567)
	at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:406)
	at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:82)
	at io.netty.channel.AbstractChannel$CloseFuture.setClosed(AbstractChannel.java:900)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.close(AbstractChannel.java:559)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.closeOnRead(AbstractNioByteChannel.java:71)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:157)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:748)`

**Comment by l3nz** [comment]
So the signature of GetStoredFile is wrong.
@edgardlopes  what were you calling? it's not apparent from the log.

**Comment by grahambrown11** [comment]
fixed from 0.4.0
